google-cloud-spanner/benchmark/bin/ycsb

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# A YCSB-like executable that can be integrated into PerfKitBenchmarker.
+#
+# Assume the following are installed on the VM.
+# gpg --keyserver hkp://keys.gnupg.net \
+#   --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 \
+#   7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+# curl -sSL https://get.rvm.io | bash -s stable --ruby
+# source $HOME/.rvm/scripts/rvm; gem install bundler
+# sudo apt-get install -y git
+# git clone https://github.com/GoogleCloudPlatform/google-cloud-ruby.git
+# cd google-cloud-ruby; bundle install
 
-bundle exec ruby ${DIR}/../ycsb.rb "${@:1}"
+source $HOME/.rvm/scripts/rvm
```

Comment:
What is users are not using RVM?

Reply:
This script assumes that the user uses RVM and is following the https://cloud.google.com/ruby/getting-started/hello-world for installing everything.

It would be nice if there is a standard way to install ruby and google-cloud-ruby package. The current installation process has many variables which is hard to map out. On the other hand, this script is for integration with PerfKitBenchmarker and we are going to make sure PerfKitBenchmarker will use RVM and put the repo at the right place.

Reply:
I'm glad to see documentation added to this script. Regarding the prerequisites, I think it would be good to add a line after line 3 stating something like:

"It is intended to be run in an <X> environment, with <a>, <b> and <c>.

These dependencies should be installed as follows:"

Basically what you already have, with just a bit more introduction, and most importantly, a list of the dependencies.

Reply:
Thx! Done.